src/CommunityToolkit.Maui/Essentials/FolderPicker/FolderPickerImplementation.macios.cs

```diff
@@ -0,0 +1,50 @@
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Primitives;
+using Foundation;
+using UIKit;
+using UniformTypeIdentifiers;
+
+namespace CommunityToolkit.Maui.Storage;
+
+/// <inheritdoc />
+public class FolderPickerImplementation : IFolderPicker
+{
+	/// <inheritdoc />
+	public async Task<Folder> PickAsync(string initialPath, CancellationToken cancellationToken)
+	{
```

Comment:
On iOS we don't need to ask for permission?

Reply:
iOS file storage is protected by default. we work in sandbox, so path permission is not needed. Also macOS asks for it automatically when call controller